src/doc/trpl/inline-assembly.md

```diff
@@ -100,6 +100,21 @@ fn main() {
 }
 ```
 
+If you would like to use real operands in this position, however,
+you are required to put curly braces `{}` around the register that
+you want, and you are required to put the specific size of the
```

Comment:
Hm, well it's definitely not *always* required. E.g. to add to the example above, this also works (the `not` indicates that the correct register size is being used and it's not just defaulting to say `eax`: the output is `fffffffffffffffd`):

```rust
#![feature(asm)]

fn main() {
    let mut test: u64;
    unsafe {
        asm!("mov $$2, %rax
              not %rax" 
             : "={ax}"(test));
    }
    println!("{:x}", test);
}
```

What's the other way in your example?